app/views/patients/index.csv.erb

```diff
@@ -0,0 +1,8 @@
+<%=
+CSV.generate do |csv|
+  csv << Patient::CSV_EXPORT_FIELDS.keys # Header line
+  @patients.each do |patient|
+    csv << Patient::CSV_EXPORT_FIELDS.values.map { |field| patient.send(field) }
```

Comment:
One alternative I had also considered is adding something like a `to_csv_entry` method on `Patient` itself that just returned a nice hash representing the given instance, thereby bypassing this entirely.  The downside though was that to print the header I'd have to do something kinda gross like get `@patients.first.to_csv_entry.keys`.